### PR TITLE
Signed-off-by: gueFDF <3237455241@qq.com>

### PR DESCRIPTION
**Pull request by gueFDF**
### What problem does this PR solve?

Issue Number: #2185  

Problem Summary:

### What is changed and how it works?

What's Changed:
Added some log information, print error code with more meaningful name.

How it Works:
Call the LibCurveErrorName(LIBCURVE_ERROR ret) function to return the specific error information corresponding to the error number.

Side effects(Breaking backward compatibility? Performance regression?):
Basically no side effects.
### Check List

- [ ] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by gueFDF** [comment]
cicheck

**Comment by gueFDF** [comment]
cicheck

**Comment by gueFDF** [comment]
cicheck

**Comment by gueFDF** [comment]
cicheck

**Comment by aspirer** [comment]
thanks for your pr. some suggestions:
1. change  the pr title to more meanfull string, such as what are you doing by this pr.
2. some file code format modifications are not quite appropriate. I have indicated in the relevant files already.

File: src/client/chunkserver_client.cpp [hunk 1]
```diff
@@ -43,19 +43,20 @@ int ChunkServerClient::UpdateFileEpoch(
...
         if (ret != 0) {
             LOG(ERROR) << "failed to init channel to chunkserver, "
-                       << "id: " << cs.peerID
-                       << ", "<< cs.internalAddr.addr_.ip
-                       << ":" << cs.internalAddr.addr_.port;
-            return -LIBCURVE_ERROR::INTERNAL_ERROR;
+                       << "id: " << cs.peerID << ", "
+                       << cs.internalAddr.addr_.ip << ":"
+                       << cs.internalAddr.addr_.port;
+            return -::INTERNAL_ERROR;
```
Comment (aspirer) [inline src/client/chunkserver_client.cpp:59]: -LIBCURVE_ERROR::INTERNAL_ERROR;

File: src/client/lease_executor.cpp [hunk 2]
```diff
@@ -29,18 +29,12 @@ using curve::common::TimeUtility;
...
-      leaseoption_(leaseOpt),
-      leasesession_(),
-      isleaseAvaliable_(true),
-      failedrefreshcount_(0),
-      task_() {}
+LeaseExecutor::LeaseExecutor(const LeaseOption &leaseOpt,
+                             const UserInfo &userinfo, MDSClient *mdsclient,
+                             IOManager4File *iomanager)
+    : fullFileName_(), mdsclient_(mdsclient), userinfo_(userinfo),
+      iomanager_(iomanager), leaseoption_(leaseOpt), leasesession_(),
```
Comment (aspirer) [inline src/client/lease_executor.cpp:36]: the original code style is better to me.

File: src/client/libcbd_ext4.cpp [hunk 3]
```diff
@@ -45,47 +45,44 @@ int cbd_ext4_init(const CurveOptions* options) {
...
     return pread(fd, buf, length, offset);
 }
 
-int cbd_ext4_pwrite(int fd, const void* buf, off_t offset, size_t length) {
+int cbd_ext4_pwrite(int fd, const void *buf, off_t offset, size_t length) {
     return pwrite(fd, buf, length, offset);
 }
 
-int cbd_ext4_pdiscard(int fd, off_t offset, size_t length) {
+int cbd_ext4_get_LIBCURVE_ERRORpdiscard(int fd, off_t offset, size_t length) {
```
Comment (aspirer) [inline src/client/libcbd_ext4.cpp:74]: fix the func name please.

File: src/client/libcurve_snapshot.cpp [hunk 4]
```diff
@@ -179,19 +180,20 @@ int SnapshotClient::CreateCloneFile(const std::string &source,
 int SnapshotClient::GetFileInfo(const std::string &filename,
                                 const UserInfo_t &userinfo, FInfo *fileInfo) {
     FileEpoch_t fEpoch;
-    LIBCURVE_ERROR ret = mdsclient_.GetFileInfo(filename, userinfo,
-                                                fileInfo, &fEpoch);
+    LIBCURVE_ERROR ret =
+        mdsclient_.GetFileInfo(filename, userinfo, fileInfo, &fEpoch);
```
Comment (aspirer) [inline src/client/libcurve_snapshot.cpp:184]: the original code style is better to me.
